### PR TITLE
created user model and migrated it

### PR DESCRIPTION
**Pull request by Tophu**
review please

### CODE REVIEW

**Comment by Sdaewa** [comment]
check check

File: db/migrate/20190226113739_devise_create_users.rb [hunk 1]
```diff
@@ -6,7 +6,11 @@ def change
       ## Database authenticatable
       t.string :email,              null: false, default: ""
       t.string :encrypted_password, null: false, default: ""
-
+      t.text :address
+      t.integer :phone_number
+      t.string :first_name
+      t.string :last_name
+      t.boolean :butler
```
Comment (Sdaewa) [inline db/migrate/20190226113739_devise_create_users.rb:13]: set default or not?

Comment (lazyrayza) [inline db/migrate/20190226113739_devise_create_users.rb:13]: Double check if you need to add a default to it!

**Review by lazyrayza — CHANGES_REQUESTED** [review]

File: db/migrate/20190226113739_devise_create_users.rb [hunk 2]
```diff
@@ -6,7 +6,11 @@ def change
       ## Database authenticatable
       t.string :email,              null: false, default: ""
       t.string :encrypted_password, null: false, default: ""
-
+      t.text :address
+      t.integer :phone_number
```
Comment (Sdaewa) [inline db/migrate/20190226113739_devise_create_users.rb]: phone number should be string

**Pull request merged**